### PR TITLE
closes #1841

### LINKED ISSUE #1841: "Smoothing" von MC A-D wird bei bei manchen älteren Presets auf 0 ms initialisiert. Es sollte immer auf 20 ms gesetzt werden (bis der User den Wert editiert), s.u.
Beispiele für Smoothing A-D = 0 ms:
Bank "NL Piano 14", Preset 019, 020, 041...056
Bank "NLPlucked 09", Preset 030, 031,
Bank "NL Guitar+ Bass", Preset 011, 016, 017

Der Initial-Wert ist zur Zeit 12.1 ms. Bitte auf 20 ms setzen (ist per "Fine"-Button auch erreichbar). Das entspricht der früheren Standard-Einstellung des globalen Smoothings.

Weil diese Parameter neu sind, können wir sie alle beim ersten Laden einer Bank nach dem Software-Update durch-initialisieren. Oder irgendwie feststellen, ob der User sie schon einmal editiert hat, und sonst auf  20 ms setzen.

### PR DESCRIPTION
**Pull request by schlenkibus**
closes #1841

### CODE REVIEW

File: projects/playground/src/parameters/MacroControlSmoothingParameter.cpp [hunk 1]
```diff
@@ -15,3 +15,9 @@ ParameterId MacroControlSmoothingParameter::getMC() const
 {
   return MacroControlsGroup::smoothingIdToMCId(getID());
 }
+
+void MacroControlSmoothingParameter::setCpValue(UNDO::Transaction* transaction, Initiator initiator,
```
Comment (hhoegelo) [inline projects/playground/src/parameters/MacroControlSmoothingParameter.cpp:19]: Why is it overridden if no different behavior is implemented?

Comment (schlenkibus) [reply projects/playground/src/parameters/MacroControlSmoothingParameter.cpp:19]: forgot to remove it after debugging

File: projects/playground/src/parameters/Parameter.h [hunk 2]
```diff
@@ -137,7 +137,7 @@ class Parameter : public UpdateDocumentContributor,
 
  protected:
   virtual void sendToLpc() const;
-  void setCpValue(UNDO::Transaction *transaction, Initiator initiator, tControlPositionValue value, bool doSendToLpc);
+  virtual void setCpValue(UNDO::Transaction *transaction, Initiator initiator, tControlPositionValue value, bool doSendToLpc);
```
Comment (hhoegelo) [inline projects/playground/src/parameters/Parameter.h:140]: Is this method still required to be virtual?

File: projects/playground/src/parameters/MacroControlSmoothingParameter.h [hunk 3]
```diff
@@ -6,5 +6,7 @@ class MacroControlSmoothingParameter : public Parameter
  public:
   using Parameter::Parameter;
   Layout *createLayout(FocusAndMode focusAndMode) const override;
+
+ public:
```
Comment (hhoegelo) [inline projects/playground/src/parameters/MacroControlSmoothingParameter.h:10]: duplicate

**Pull request merged**